src/components/PollCard.tsx

```diff
@@ -0,0 +1,157 @@
+import { LOCALE } from '@/constants/enum';
+import { env } from '@/constants/env';
+import { IMAGE_THEME, THEME_CONFIG } from '@/constants/theme';
+import { getPollTimeLeft } from '@/helpers/getPollTimeLeft';
+import { indexToLetter } from '@/helpers/indexToLetter';
+import { PollTheme } from '@/types';
+import { ChoiceDetail, Poll } from '@/types/api';
+
+export interface PollCardProps {
+    poll: Poll;
+    theme: IMAGE_THEME;
+    locale: LOCALE;
+}
+
+interface VoteButtonProps {
+    text: string;
+    theme: PollTheme;
+}
+
+interface VoteResultProps {
+    choice: ChoiceDetail;
+    theme: PollTheme;
+}
+
+function VoteButton({ text, theme }: VoteButtonProps) {
+    return (
+        <div
+            style={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+                color: theme.optionTextColor,
+                height: 40,
+                width: '100%',
+                borderRadius: 10,
+                border: `1px solid ${theme.optionTextColor}`,
+                fontSize: 16,
+                fontWeight: 'bold',
+            }}
+        >
+            {text}
+        </div>
+    );
+}
+
+function VoteResult({ choice, theme }: VoteResultProps) {
+    return (
+        <div
+            style={{
+                position: 'relative',
+                display: 'flex',
+                height: 40,
+            }}
+        >
+            <div
+                style={{
+                    display: 'flex',
+                    width: choice.percent ? `${choice.percent}%` : '10px',
+                    position: 'absolute',
+                    height: '100%',
+                    borderRadius: 10,
+                    backgroundColor: choice.is_select ? theme.optionSelectedBgColor : theme.optionBgColor,
+                }}
+            />
+            <div
+                style={{
+                    position: 'absolute',
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                    paddingLeft: 20,
+                    height: '100%',
+                    width: '100%',
+                    fontSize: 16,
+                    fontWeight: 'bold',
+                    color: choice.is_select ? theme.optionSelectedTextColor : theme.optionTextColor,
+                }}
+            >
+                <span
+                    style={{
+                        display: 'flex',
+                        alignItems: 'center',
+                        gap: 8,
+                    }}
+                >
+                    <span style={{ display: 'flex' }}>{choice.name}</span>
+                    {choice.is_select ? (
+                        <img
```

Comment:
Use `<Image />`?

Reply:
we cant use Image here. This component works in frame.js which doesn't work with Image